app/models/user.rb

```diff
@@ -5,6 +5,6 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
   
   enum role: [:sender, :traveler]
-  has_one :journey
+  has_many :journey
```

Comment:
it should be `has_many :journeys` not `has_many :journey`

Reply:
typing mistake.fixed